content/docs/introduction/technical-preview-free-tier.md

```diff
@@ -20,6 +20,10 @@ The Neon Free Tier has the following limits:
 - 100 hours of _compute active time_ per month (total) on non-primary branches. When you reach the limit, compute endpoints assigned to [non-primary branches](/docs/reference/glossary#non-primary-branch) are unavailable until the limit resets at the beginning of the next month.
 - A point-in-time restore window of 7 days of _reasonable usage_.
 
+<Admonition type="note">
+On February 24, 2023, Neon removed the limit on the number of compute endpoints per project. The Free Tier now offers a compute endpoint for every branch and 100 hours of compute endpoint _active time_, as described above. The _active time_ limit is in effect for Neon projects created after this date. The limit will not be applied to Neon projects created before this date until April 1, 2023, to provide enough time for users to adjust their usage or upgrade to a paid plan. Neon launched paid plans in March, 2023. For information about our paid plans, refer to our [Pricing](https://neon.tech/pricing) page.
```

Comment:
It's good to have the correct date here, but this note almost duplicates the other note (see lines 10-12 in this file). Also, do we even need this note at all? I mean, it's already 5th of April, and old users are limited too

Reply:
Good point. The *Read more* link is gone from the Dashboard. Let's remove it entirely.